### DISCUSSION TITLE
Does PinInput support autocomplete="one-time-code"?

### DISCUSSION THREAD

**Discussion by moroshko** [chakra-ui/chakra-ui#2075] [General] [answered]
The [`PinInput` docs](https://next.chakra-ui.com/docs/form/pin-input#autofill-and-copy--paste) mention "autofill", but don't really say anything about it.
Can [this autofill magic](https://twitter.com/IMAC2/status/1302942695980183556) work with the `PinInput` component?

**Comment by brunoalano** [comment]
Any update on this?

**Comment by dodas** [comment, marked as answer]
This has been implemented in #3095.
Just add the `otp` prop to the `PinInput` component.
